### DISCUSSION TITLE
Linking to slides by name/ID

### DISCUSSION THREAD

**Discussion by benui-dev** [hakimel/reveal.js#3308] [Q&A] [answered]
I couldn't find any mention of this on the documentation, is there a way to allow linking to particular slides by a named ID rather than slide index?

**Comment by hakimel** [comment, marked as answer]
You can name slides using `id` attributes and link to them with `href="#/<id>"`. Here's an example from the docs:

```html
<section>
	<a href="#/grand-finale">Go to the last slide</a>
</section>
<section>
	<h2>Slide 2</h2>
</section>
<section id="grand-finale">
	<h2>The end</h2>
	<a href="#/0">Back to the first</a>
</section>
```

More info here: https://revealjs.com/links/

**Comment by lechten** [comment]
Issue #2276 questions that above link format, which results in broken links (e.g., search engines do not understand that an id is supposed to be referenced). Links without the `/` after `#` seems to work partially...

**Comment by peter-lyons-kehl** [comment]
If you use the Markdown plugin, apply [Markdown > Element Attributes](https://revealjs.com/markdown/#element-attributes) to every slide involved.
```
<!-- .slide: id="grand-final" -->
```

Multiple slides within the same `<script type="text/template">...</script>` or within the same Markdown file are separated by `---`.
